src/modules/customer/containers/HappyUserPage/components/formFields/CheckboxGroup.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import messages from '../WizardForm/messages';
+
+import '../../components/WizardForm/WizardForm.css';
+
+class CheckboxGroup extends React.PureComponent {
+  checkboxGroup() {
+    let { options, input } = this.props;
+
+    return options
+      .map((option, index) => {
+        return (
+          <div className="wizard-form__checkbox" key={index}>
```

Comment:
Try to never ever use index as key. Only in very simple readonly lists. In functional components that will sooner or later lead to bugs, hard to fix. In this case use something like ``checkbox-group-option-${option.value}``